scripts/load-gotland.sh

```diff
@@ -376,6 +376,24 @@ while [ "$year" -le 2019 ]; do
 	year=$(( year + 1 ))
 done | psql --quiet
 
+# Fix up the body fat data
+cat <<-END_SQL
```

Comment:
For me I get a print out on the screen data is unchanged?

Reply:
Duh. Sorry.